prisma/seeds/seed.ts

```diff
@@ -143,6 +143,49 @@ async function deliveryOptionMigration() {
   );
 }
 
+// Part 1: Notifications v2 migration
+// Can be removed after the migration is completed
+async function emailNotificationsUsersMigration() {
+  const templates = await prisma.template.findMany({
+    where: { notificationsInterval: { not: null } },
+    select: {
+      id: true,
+      users: { select: { id: true } },
+    },
+  });
+
+  await Promise.all(
+    templates.map((template) =>
+      prisma.template.update({
+        where: { id: template.id },
+        data: {
+          notificationsUsers: {
+            connect: template.users.map((u) => ({ id: u.id })),
+          },
+        },
+      })
+    )
+  );
+
+  console.log(
+    `${templates.length} were migrated for update email notifications users that have a non null notificationsInterval`
+  );
+}
+
+// Part 2: Notifications v2 migration
```

Comment:
The notificationsInterval setting could instead be removed but if so this should also be removed in the UI and a hard coded value used consistently.